moveToWishList.js

```diff
@@ -1,48 +1,80 @@
-	* 
-	*/
+Timing Issues:
+	Edit the number in every "setTimeout" command, in miliseconds, to whatever works for you.
+		(Note; if possible, in future iterations a variable with the delay timer could be used. For now, each command would need editing.)
+		Each timer is in miliseconds, longer page loading will need you to increase this. Default delays *should* be ok.
+		
+Possible future versions could include:
+	* A second timing option (Timer for executing the second action after the first, then another timer for the delay before starting the first action again for the next item.)
+	* Delete function with a toggle, in case you want to delete everything after moving all items.
+	* Fix retry if wishList hadn't loaded yet.
+*/
+			
+var wishListID = "XXXXXXXXXXXXX"; // Change X's to your wishlist ID, read Usage section on how to find.
+// var actionTiming = 5000; // Delay in ms, change to your needs - to be fixed?
+
+// Set X items
+function countItems() {
 	var savedItems = document.getElementById('sc-saved-cart-list-caption-text').getAttribute('data-saved-item-quantity');
-	var cartItems = document.getElementById("nav-cart-count").innerText;
-	var totalItems = savedItems + cartItems;
-	
-	//Loop until finished
-	while(totalItems > 0) {
-	//Check that our saved items hasn't reached zero index yet
-	if (totalItems.length > 0) {
-		//Check that there is a wish list option for items
-		if(query.length) {
-			query[0].click();
-		}
-		//Check for the proper registry list (MAKE SURE TO CHANGE QUERY2 VARIABLE)
-		if (query2.length) {
-			query2[0].click();
-		}
-		//The list isn't zero but no wish list option was found, wait for the page to refresh more saved items
-		else if (!query.length) {
-		setTimeout(deleteSavedItems, 500);
-		}
+	console.log(savedItems);
+	actionOne(savedItems);
+}
+
+// Action 1/2 - Click "Add to list", open wishList dropdown.
+function actionOne(savedItems, counter) {
+	// Setting up a counter so we can tell this script to stop trying.
+	if (counter === undefined) {
+		var counter = 0;
 	}
-	//Quickly moves the items that are present
-	if (query.length > 1) {
-		setTimeout(deleteSavedItems,35);
+	// find "Add to list", then click it.
+	var addList = document.querySelectorAll("#sc-saved-cart input[value='Add to list']");
+	console.log(addList);
+	if (savedItems !== 0 && counter != 10) { // If the counter (mentioned below) reaches 10 we will stop attempting.
+		addList[0].click();
+		console.log('Waiting...');
+		setTimeout(actionTwo,500,savedItems, counter); // 500ms delay
+		// Add to list variable gets reset, just the way we like it.
 	}
-	savedItems--;
-	}	
+// Action 2/2 - Add to wishList
+function actionTwo(savedItems, counter) {
+	console.log('Returning to next action');
+	// find wishlist item in dropdown that I HOPE loaded.
+	console.log('Setting wishList var');
+	console.log("#cldd-add-item-declarative-" + wishListID + " a");
+	wishList = document.querySelectorAll("#cldd-add-item-declarative-" + wishListID + " a");

```

Comment:
I just noticed I have a typo.
Should be `var wishList =`, not `wishList =`

Reply:
Fixed, and appears to have auto-updated this pull request...    
On top of learning some JavaScript, I'm also learning how GitHub Pull Requests work